src/components/EditImageModal/EditImageModal.test.jsx

```diff
@@ -72,23 +76,37 @@ let editImageModal;
 
 describe('EditImageModal', () => {
   beforeEach(() => {
+    querySelector = global.document.querySelector;
+    parentElement = global.document.createElement('div');
+    global.document.body.appendChild(parentElement);
+    global.document.querySelector = () => parentElement;
+    mockPortal.init();
```

Comment:
Why does the EditImageModal test need the mockPortal.init(), but the AssetsTable one does not?

Reply:
Enzyme actually supports portals when using `mount()` now, it just doesn't work with `shallow()`. And, `AssetsTable.test.jsx` uses `mount()`.